### ISSUE TITLE
Accessibility: Ctrl+Down/UpArrows does not work in Tree find control

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#180221]
CC @meganrogge @isidorn

1. Open File Explorer (Ctrl+Shift+E).

1. Ctrl+F to open Tree find control and type search term.

1. Press Ctrl+DownArrow to see if you can jump to the search result: not working.

1. In the search result, press Ctrl+UpArrow to see if you can go back to search input: not working.

**Comment by meganrogge** [comment]
this has the same root cause as https://github.com/microsoft/vscode/issues/179123

`list.scrollUp` is activated in this case

**Comment by meganrogge** [comment]
This would require adding something analogous to `search.action.focusSearchFromResults` and overriding the `list.scrollUp` similar to what we did for https://github.com/microsoft/vscode/issues/179123

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by gjsjohnmurray** [comment]
@meganrogge maybe to stop the bot interfering this needs its milestone changing to Backlog?

**Comment by joaomoreno** [comment]
> Down Arrow or Enter will take you from the widget to the first match.
> 
> Ctrl/Cmd + F will take you back to the widget.

@meganrogge Isn't this enough? We have keyboard shortcuts to jump to results and back to the search widget. Do we need anything else?

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
